ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/block/BlockManager.java

```diff
@@ -145,23 +144,40 @@ private void importBlockIgnoringResult(final SignedBeaconBlock block) {
             result -> {
               if (result.isSuccessful()) {
                 LOG.trace("Imported block: {}", block);
-              } else if (result.getFailureReason() == FailureReason.UNKNOWN_PARENT) {
-                // Add to the pending pool so it is triggered once the parent is imported
-                pendingBlocks.add(block);
-                // Check if the parent was imported while we were trying to import this block and if
-                // so, remove from the pendingPool again and process now We must add the block to
-                // the pending pool before this check happens to avoid race conditions between
-                // performing the check and the parent importing.
-                if (recentChainData.containsBlock(block.getParentRoot())) {
-                  pendingBlocks.remove(block);
-                  importBlockIgnoringResult(block);
-                }
-              } else if (result.getFailureReason() == FailureReason.BLOCK_IS_FROM_FUTURE) {
-                futureBlocks.add(block);
               } else {
-                LOG.trace(
-                    "Unable to import block for reason {}: {}", result.getFailureReason(), block);
-                dropInvalidBlock(block);
+                switch (result.getFailureReason()) {
+                  case UNKNOWN_PARENT:
+                    // Add to the pending pool so it is triggered once the parent is imported
+                    pendingBlocks.add(block);
+                    // Check if the parent was imported while we were trying to import this block
+                    // and if
+                    // so, remove from the pendingPool again and process now We must add the block
+                    // to
+                    // the pending pool before this check happens to avoid race conditions between
+                    // performing the check and the parent importing.
+                    if (recentChainData.containsBlock(block.getParentRoot())) {
+                      pendingBlocks.remove(block);
+                      importBlockIgnoringResult(block);
+                    }
+                    break;
+                  case BLOCK_IS_FROM_FUTURE:
+                    futureBlocks.add(block);
+                    break;
+                  case FAILED_EXECUTION_PAYLOAD_EXECUTION_SYNCING:
+                    LOG.error("Unable to import block: Execution Client is still syncing");
```

Comment:
I'm leaning towards making this a `WARN`.  Things are probably working correctly but just require more time and the user doesn't need to intervene at this stage (vs the communication error below which shouldn't happen).